src/compas_viewer/scene/bufferobject.py

```diff
@@ -206,7 +207,8 @@ def __init__(
         self.linewidth = 1.0 if linewidth is None else linewidth
         self.opacity = 1.0 if opacity is None else opacity
         self.doublesided = True if doublesided is None else doublesided
-        self.is_visible = True if is_visiable is None else is_visiable
+        self.show = True if is_visiable is None else is_visiable
+        self._is_locked = False if is_locked is None else is_locked
```

Comment:
were we not going to remove this?

Reply:
yes we are, but it would be better to do that in another PR, cuz this attribute is a bit everywhere, this one here is added temporarily to make the form work